handler_error.go

```diff
@@ -212,32 +209,6 @@ func (e *ErrorHandler) HandleError(w http.ResponseWriter, r *http.Request, errMs
 		w.Header().Add("Connection", "close")
 	}
 
-	var obfuscated string
```

Comment:
It seems like this part was completely removed, without being replaced by `GetIPFromRequest`. Even if `clientIP` not used anywhere, it still miss error logging, in case if `RemoteAddr` has issues.